sites/partners/src/components/listings/PaperListingForm/sections/AdditionalDetails.tsx

```diff
@@ -23,25 +29,47 @@ const AdditionalDetails = (props: AdditionalDetailsProps) => {
         heading={t("listings.sections.additionalDetails")}
         subheading={t("listings.sections.additionalDetailsSubtitle")}
       >
+        <Grid.Row columns={1}>
+          <Grid.Cell>
+            <FieldGroup
+              name={"selectedRequiredDocuments"}
+              groupLabel={
+                <span>
+                  {t("listings.requiredDocuments")}
+                  {jurisdiction && <span className={styles["asterisk"]}>{` ${"*"}`}</span>}
```

Comment:
issue: the asterisk should only be shown here if the "requiredDocumentsList" is in the jurisdiction's "requiredListingFields". Right now you are just look if a jurisdiction is selected or not.